src/commands/build.rs

```diff
@@ -1,16 +1,84 @@
 pub async fn build(config: &fpm::Config) -> fpm::Result<()> {
     tokio::fs::create_dir_all(config.build_dir()).await?;
 
-    let (documents, config) = fpm::get_documents_with_config(config).await?;
+    let translation_snapshots = if let Some(ref original) = config.package.translation_of.as_ref() {
+        let translation_config = config.read_translation().await?;
+        let documents = config
+            .get_translation_documents()
+            .await?
+            .iter()
+            .map(|v| {
+                (
+                    v.get_id()
+                        .replace(&format!(".packages/{}/", original.name), ""),
```

Comment:
Use proper file API for this. You are looking for parent folder, or just the file name without parent etc. Use functions, give them name so people have clues of what is happening.